### PR TITLE
Removes HOS parade jackets

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

Nukes the old HOS parade jackets, because of tgstation/tgstation#60414

## Why It's Good For The Game

Unnecessary to have.

## Changelog
:cl:
del: Removed HOS Parade jackets.
/:cl:

### CODE REVIEW

**Comment by axietheaxolotl** [comment]
guh

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -81,9 +81,9 @@
 	new /obj/item/clothing/suit/toggle/armor/hos/hos_formal(src)
 	new /obj/item/cartridge/hos(src)
 	new /obj/item/radio/headset/heads/hos(src)
-	new /obj/item/clothing/suit/armor/hos/parade/female(src) // SKYRAT EDIT CHANGE - ORIGINAL: new /obj/item/clothing/under/rank/security/head_of_security/parade/female(src)
+	// new /obj/item/clothing/suit/armor/hos/parade/female(src) // SKYRAT EDIT CHANGE - ORIGINAL: new /obj/item/clothing/under/rank/security/head_of_security/parade/female(src) // SKYRAT EDIT REMOVAL - OVERRIDDEN BY https://github.com/tgstation/tgstation/pull/60414
```
Comment (ZephyrTFA) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]:
```suggestion
```

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -81,9 +81,9 @@
...
 	new /obj/item/clothing/under/rank/security/head_of_security/parade(src)
-	new /obj/item/clothing/suit/armor/hos/parade(src) // SKYRAT EDIT ADDITION
+	// new /obj/item/clothing/suit/armor/hos/parade(src) // SKYRAT EDIT ADDITION // SKYRAT EDIT REMOVAL - OVERRIDDEN BY https://github.com/tgstation/tgstation/pull/60414
```
Comment (ZephyrTFA) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -32,13 +32,16 @@
...
+// SKYRAT EDIT REMOVAL - OVERRIDDEN BY https://github.com/tgstation/tgstation/pull/60414
+/*
 /obj/item/clothing/suit/armor/hos/parade/female
 	name = "head of security's female parade jacket"
 	desc = "A luxurious jacket for the head of security, woven in a deep red. This one comes with white trousers. On the lapel is a small pin in the shape of a deer's head."
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
 	icon_state = "hos_parade_fem"
 	inhand_icon_state = "hos_parade_fem"
+*/
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/clothing/suits/armor.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm [hunk 4]
```diff
@@ -21,6 +23,7 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	cold_protection = CHEST|GROIN|ARMS
 	heat_protection = CHEST|GROIN|ARMS
+*/
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm]: Cant also code suggest here because web review is shit

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]
You dont need to do skyrat edit comments in modular files;
and you dont need to comment out skyrat additions, you can just yeet them

**Pull request merged**